docs/examples/message-box/alert.vue

```diff
@@ -8,6 +8,7 @@ import type { Action } from 'element-plus'
 
 const open = () => {
   ElMessageBox.alert('This is a message', 'Title', {
+    // autofocus: true,
```

Comment:
Why add comments here?

Reply:
It is the first example of a message-box, and since we changed the behavior, I want this property to be quickly discoverable when the user views the example.